module.lua

```diff
@@ -109,43 +109,40 @@ local _TEAM = {
 		["Eyeground#0000"] = "FR",
 		["Faz_x#0010"] = "PL",
 		["Flare#0010"] = "NL",
-		["Flashpoint#0015"] = "EN",
-		["Frankenshtein#0010"] = "RU",
+		["Frankenshtein#0095"] = "RU",
 		["Goondad#0020"] = "EN",
 		["Grapeup#0020"] = "IL",
 		["Grimmaro#0095"] = "EN",
 		["Harshy#0095"] = "EN",
-		["Hispano#0010"] = "ES",
 		["Hosual#0000"] = "PL",
 		["Icewolfbob#0095"] = "EN",
-		["Ihaya#0010"] = "RU",
-		["Ikke#0015"] = "BR",
+		["Ihaya#0095"] = "RU",
 		["Ildubbio#0010"] = "RO",
 		["Irishcow#0095"] = "EN",
 		["Iupi#0010"] = "BR",
 		["Jackgt20#0010"] = "ES",
-		["Jacob#0010"] = "EN",
-		["Jealy#8939"] = "EN",
+		["Jacob#0095"] = "EN",
+		["Jealy#0010"] = "EN",
 		["Jefitou#0010"] = "ES",
-		["Jerry#0015"] = "RU",
+		["Jerry#0095"] = "RU",
 		["Jiaxian#0000"] = "MY",
 		["Jiro#0010"] = "EN",
 		["Jordy#0010"] = "NL",
-		["Karbi#0010"] = "ES",
+		["Karbi#0095"] = "ES",
 		["Katburger#0015"] = "EN",
 		["Kathryn#8850"] = "EN",
 		["Katow#0020"] = "FR",
 		["Kim#0010"] = "EN",
 		["Kiwrimai#0015"] = "BR",
 		["Kloure#0010"] = "EN",
 		["Kurt#0010"] = "BR",
-		["Lament#0015"] = "PL",
-		["Lapinprince#0015"] = "EN",
```

Comment:
Idk if he needs to be here, dont know if he's still fc on this account.

Reply:
Yes, they are. Add Lapinprince#0015 back.

@edit: I'll add it myself